Maskbook/Scene/App/RedPacket/Model/LuckDropHistory+BlockChainNetwork.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+
+extension BlockChainNetwork {
+    var startBlock: Int? {
```

Comment:
For now `Int` is enough to store blockheight,
But I think Int64 is safer for some chains witch have a larger blockheight

Reply:
StartBlock won't change, if there is a new chain to support, which is greater than Int.max, will use `BigUInt` instead.

Reply:
I think good code should avoid future error.
We can’t guarantee someone apply that `if there is a new chain to support, which is greater than Int.max, will use BigUInt instead. `

Reply:
For current implement, all the startBlock values are fitting `Int` type. And when adding new case, if the value is overflow, the compiler will complain and we can do refactor according to the compiler error, this is important.

Reply:
<img width="399" alt="Screen Shot 2022-04-20 at 14 25 06" src="https://user-images.githubusercontent.com/6863518/164164000-b80d9d5a-acbb-4a2f-bb9d-81f73b39077a.png">
For above code the compiler don't give a warning or an error.
But it will overflow in 32 bit device.